tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java

```diff
@@ -14,6 +14,8 @@
 
 	public double getPP(double acc, int combo, int misses);
 
+	public double getPPForHitPoints(int x100, int x50, int combo, int misses);
```

Comment:
altho tbh, I dont think my idea of just calling it `getPP` and let the parameters speak for itself isnt such a bad idea